ivadomed/main.py

```diff
@@ -210,6 +210,18 @@ def run_command(context, n_gif=0, thr_increment=None):
         return best_training_dice, best_training_loss, best_validation_dice, best_validation_loss
 
     if thr_increment:
+        # LOAD DATASET
+        # Get Validation dataset
+        ds_valid = imed_loader.load_dataset(**{**loader_params,
+                                               **{'data_list': valid_lst, 'transforms_params': transform_valid_params,
+                                                  'dataset_type': 'validation'}}, device=device,
+                                            cuda_available=cuda_available)
+        # Get Training dataset
+        ds_train = imed_loader.load_dataset(**{**loader_params,
+                                               **{'data_list': train_lst, 'transforms_params': transform_train_params,
```

Comment:
@andreanne-lemay --> needs transform_valid here, otherwise DA active! Will change this